python/ray/scripts/scripts.py

```diff
@@ -1113,12 +1113,17 @@ def statistics(address):
     required=False,
     type=str,
     help="Override the address to connect to.")
-def memory(address):
+@click.option(
+    "--redis_password",
+    required=False,
+    type=str,
```

Comment:
Can you provide a default? and make sure this defaults to 
`ray_constants.REDIS_DEFAULT_PASSWORD` (see worker.py)

Reply:
Added! 😄